core/agents/developer.py

```diff
@@ -262,12 +262,19 @@ async def ask_to_execute_task(self) -> bool:
 
         :return: True if the task should be executed as is, False if the task is skipped or edited
         """
+        buttons = {"yes": "Yes", "edit": "Edit Task"}
+        if len(self.current_state.tasks) > 1:
+            buttons["skip"] = "Skip Task"
```

Comment:
We intentionally don't want to allow skipping the first task?

Reply:
this will show the "Skip task" button to the user only if Epic has more than 1 task. If there is only 1 task in epic we dont want to allow user to skip because then epic would be marked as finished/done and nothing was implemented. Makes no sense to skip a task if there is only one task in the whole epic